utils/guidelines_xslt/odd2html/htmlFile.xsl

```diff
@@ -61,6 +61,8 @@
                             href="{$cssPath}css/mei-website.css" />
                         <link rel="stylesheet" media="screen" type="text/css"
                             href="{$cssPath}css/mei-screen.css" />
+                        <link rel="stylesheet" media="screen" type="text/css"
+                            href="{$cssPath}css/guidelines.css" />
```

Comment:
What's the difference of this new `guidelines.css` file to `mei-screen.css`? Would it make sense to have those declarations there, too?

Reply:
Also, do we need to handle those val and att tags in the printed guidelines as well (cc @kepper?)?

Reply:
My idea was to separate rules to show what is specific for the guidelines. I'd leave `mei-print.css` to @kepper, as I don't know anything about this and how to test it.

Reply:
I understand your intention, but is the function of mei-screen.css not the same as your intended function for guidelines.css (both having now rules specific for the guidelines)? But maybe I do not fully understand what mei-screen.css does exactly.

Reply:
I guess these two lines are now obsolete, too, since the rules were re-integrated with mei-screen.css